packages/@okta/vuepress-site/docs/guides/pwd-optional-new-sign-up-email/main/index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Sign up for new account with email only
+---
+
+<div class="oie-embedded-sdk">
+
+<ApiLifecycle access="ie" /><br>
+
+Learn how to use the Embedded SDK to integrate password optional sign up in your app.
```

Comment:
```suggestion
Learn how to use the Embedded SDK to integrate password-optional sign-up flows in your app.
```

Reply:
Issue fixed in PR: https://github.com/okta/okta-developer-docs/pull/3435